### PR TITLE
Patch CToken Oracle Calculation

### PR DESCRIPTION
**Pull request by felix2feng**

### CODE REVIEW

File: contracts/meta-oracles/oracles/CTokenOracle.sol [hunk 1]
```diff
@@ -41,21 +41,32 @@ contract CTokenOracle is
...
+
+    // Underlying Asset Full Unit
+    uint256 public underlyingFullUnit;
+
     /* ============ Constructor ============ */
 
     /*
      * @param  _cToken             The address of Compound Token
      * @param  _underlyingOracle   The address of the underlying oracle
+     * @param  _underlyingFullUnit The full unit of the underlying asset
```
Comment (richardliang) [inline contracts/meta-oracles/oracles/CTokenOracle.sol]: Missing `cTokenFullUnit`

File: package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     "truffle-test-integrations": "truffle test `find transpiled/test/integrations -name '*.spec.js'`",
     "truffle-test-contracts": "truffle test `find transpiled/test/contracts -name '*.spec.js'`",
     "prepublishOnly": "yarn dist",
-    "flatten": "truffle-flattener contracts/[xxxx]/[xxx].sol"
+    "flatten": "truffle-flattener contracts/meta-oracles/oracles/CTokenOracle.sol"
```
Comment (richardliang) [inline package.json]: Keep it like before?

File: contracts/meta-oracles/oracles/CTokenOracle.sol (hunk 1 above)
Comment (felix2feng) [reply contracts/meta-oracles/oracles/CTokenOracle.sol]: +1

File: package.json (hunk 2 above)
Comment (felix2feng) [reply package.json]: +1

**Pull request merged**